[keymap] Add new keymap for TADA68

rys opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

<!--- This template is entirely option and can be removed, but is here to help both you and us. -->
<!--- This text and anything on lines wrapped like this one will not show up in the final text. This text is to help us and you. -->

## Description
<!--- Describe your changes in detail -->
Adds a new layout for the TADA68. It's similar to my `rys` layout where the main feature is `KC_LALT` and `KC_LGUI` swapped to work better on macOS, and it has tested fine on a brand new TADA68 build I'm giving away to someone. I'm starting the layout for him to work on and customise later!

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation


## Issues Fixed or Closed by this PR

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

noroadsleft commented on keyboards/tada68/keymaps/mattgemmell/keymap.c:
```diff
@@ -0,0 +1,50 @@
+#define _BL 0
+#define _FL 1
+
+#define _______ KC_TRNS
```
QMK defines this keycode alias automatically. Line 6 can be deleted.

https://github.com/qmk/qmk_firmware/blob/7c63c0a95075f6fb4dd94d020ceb5816e6bcca2c/quantum/quantum_keycodes.h#L29-L31

rys replied:
Resolved in 6a75419

drashna commented on keyboards/tada68/keymaps/mattgemmell/keymap.c:
```diff
@@ -0,0 +1,50 @@
+#define _BL 0
+#define _FL 1
+
+#define _______ KC_TRNS
```
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

rys replied:
Resolved in 6a75419

rys commented:
I fixed the unnecessary `KC_TRNS` define in this keymap and also in my other `rys` keymap, and renamed the PR to make it clear it's a new keymap I'm adding, not a new layout (sorry for the confusion!). Thanks for the feedback!
